src/components/PersonaDescription/PersonaDescription.tsx

```diff
@@ -0,0 +1,24 @@
+import { Avatar, Textarea, Typography } from "@material-tailwind/react";
+
+interface PersonaDescriptionProps {
+  persona: {
+    imageURL: string,
+    name: string,
+    description: string,
+  }
+}
+
+function PersonaDescription({ persona }: PersonaDescriptionProps) {
+  const { name, imageURL, description } = persona
+  return (
+    <>
+      <Avatar src={imageURL} alt="avatar" />
+      <div>
+        <Typography variant="h6">{name}</Typography>
```

Comment:
This is more of a neutral comment, but I am curious whether the material UI variant for text is semantic (for accessibility). Definitely something to keep an eye on as we add more content to the site to make sure we are giving a good experience to anyone using a screen reader.

Reply:
I did some research and updated 'h6' as 'lead'.  

The docs don't explicitly state what 'lead' does, but it renders a <p> with larger text, which is what we're looking for.    Bootstrap has a 'lead' class that 'Makes a paragraph stand out', which seems to be what MaterialTailwind's 'lead' variant accomplishes.